client/components/wcc-settings-form/settings-group.js

```diff
@@ -2,42 +2,54 @@ import React, { PropTypes } from 'react';
 import CompactCard from 'components/card/compact';
 import FormSectionHeading from 'components/forms/form-section-heading';
 import SettingsItem from './settings-item';
-import { connect } from 'react-redux';
-import * as FormActions from 'state/form/actions';
-import { bindActionCreators } from 'redux';
 import SaveForm from 'components/save-form';
 
-const renderGroupItems = ( items, schema, storeOptions ) => {
-	return (
-		items.map( item => (
+const SettingsGroup = ( {
+	group,
+	schema,
+	storeOptions,
+	form,
+	saveForm,
+	dismissSuccess,
+	errors,
+} ) => {
+	const renderSettingsItem = ( item ) => {
+		const key = item.key ? item.key : item;
+		const itemErrors = errors.length ? errors.filter( ( error ) => {
+			const path = error.split( '.' );
+			return path[0] === key;
+		} ).map( ( path ) => {
+			if ( 0 === path.indexOf( key + '.' ) ) {
+				return path.substr( key.length + 1 );
+			}
+			return path;
+		} ) : false;
+
+		return (
 			<SettingsItem
-				key={ item.key ? item.key : item }
```

Comment:
`const key = item.key ? item.key : item;`

This ☝️  was added above the usage of the spread. As I understand it, this should function identically to the old code.